spec/support/shared_examples_shared_examples_policy_machine_storage_adapter.rb

```diff
@@ -200,7 +200,7 @@
     it 'stores the association' do
       policy_machine_storage_adapter.add_association(@ua, Set.new([@r, @w]), @oa, 'some_policy_machine_uuid1')
       assocs_with_r = policy_machine_storage_adapter.associations_with(@r)
-      assocs_with_r.size == 1
+      assocs_with_r.size.should eq 1
```

Comment:
It is, but it's all over the place in this gem, it'll be a separate story to convert all of it to expect.